verification/cbmc/proofs/run-cbmc-proofs.py

```diff
@@ -163,6 +163,13 @@ def run_build(litani, jobs):
     cmd = [str(litani), "run-build"]
     if jobs:
         cmd.extend(["-j", str(jobs)])
+    # Set a limit on the number of concurrent jobs based on CPUs
+    # https://github.com/aws/aws-encryption-sdk-c/pull/673
+    else:
+        factor   = 0.75
```

Comment:
I appreciate you giving a good name to this constant.  I'm uncertain about whether this should be a top-level comment or hidden here in the code.